manifests/base/alertmanager/alert_rules.yaml

```diff
@@ -37,7 +37,7 @@ data:
           annotations:
             summary: There is a policy report with severity of critical.
             description: "The policy: {{ $labels.policy }} has a severity of critical on cluster: {{ $labels.cluster }}"
-          expr: sum(acm_policyreport_info{severity="critical"}) by (cluster, category, clusterID, policy, severity) > 0
+          expr: sum(policyreport_info{severity="critical"}) by (managed_cluster_id, category, clusterID, policy, severity) > 0
```

Comment:
correct. Thanks @xrajesh . We missed during the original review that the cluster label - added by Observability - will always be local cluster and will be useless. The managed_cluster_id is real identity of the cluster impacted. Then is ClusyerID also redundant? Will it always have the same Hub Server Id since this metric is always being generated at the Hub. @marcolan018  thoughts?

Reply:
Wouldn't we need the `cluster` label for the managed cluster? Will the managed cluster id contain the managed cluster name?